### ISSUE TITLE
[Feature]: [Entity card] extend the info from another entity

### ISSUE THREAD

**Issue by MaJerle** [piitaya/lovelace-mushroom#1477]
### Requirements

- [X] I have updated Mushroom to the latest available version
- [X] I did a search to see if there is a similar issue or if a pull request is open.

### Is your feature request related to a problem?

N/A

### Describe the solution you'd like

Entity has a primary and secondary info. As a primary, I'd like to have to keep its name, but for secondary, I'd like to show to state and then use the second entity of the same device that contains the battery level in percentage.

For instance:
Primary info: `Motion sensor`
Secondary info: `Detected, 100%`

or for sensor
Primary info: `Temperature`
Secondary info: `24.5°C, battery 79%`

where the battery level is a state from another entity.

**Comment by piitaya** [comment]
Agree, the template is made for this particular use case. I'm closing this issue has it can be done with template card.
